src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -601,6 +606,30 @@ public AdsBaseObject triggerAdSlot(String slotName) {
     return this;
   }
 
+  private void triggerFMR() {
+    scrollToPosition(By.cssSelector("#wikia-recent-activity"));
+
+    doUntilElementVisible(By.cssSelector(AdsContent.getSlotSelector(AdsContent.FLOATING_MEDREC)), () -> {
+      jsActions.scrollBy(0, 100);
+      wait.forX(Duration.ofSeconds(1));
```

Comment:
If we will have defect like lack of FLOATING_MEDREC test never ends we should implement like check max 5 times

Reply:
Good point! Thanks :)